CHANGES.txt

```diff
@@ -44,6 +44,10 @@ Changes
 Fixes
 =====
 
+ - Fixed an issue that resulted in rows could not be queried by primary keys
+   when the order of the primary key columns on insert differs from the order
+   of the primary keys on the table definition.
```

Comment:
Should we clarify that this doesn't fix anything that was already inserted incorrectly before the fix?